spinnaker-monitoring-daemon/spinnaker-monitoring/datadog_service.py

```diff
@@ -146,8 +148,16 @@ def read_param(param_name, config_object):
   if host is None and datadog_host is None:
     host = ''
   host = socket.getfqdn(host) if host is not None else datadog_host
-
-  return DatadogMetricsService(api_key=api_key, app_key=app_key, host=host)
+  static_datadog_tags = []
+  for tag_pair in filter(None, os.getenv('DATADOG_TAGS', '').replace(' ', '').split(',')):
+    if ':' in tag_pair:
+      k, v = tag_pair.split(':')
+      static_datadog_tags.append({'key': k, 'value': v})
+    else:
+      static_datadog_tags.append({'key': tag_pair, 'value': ''})
```

Comment:
what do empty tag values indicate? why would you have an empty tag?

Reply:
Technically, datadog allows both k/v pairs and singleton values to be tags.  The way the code is set up assumes k/v pairs, so I'm shoehorning singleton tags into that setup.  I personally never use them (and I doubt many other people do, either), but I figured it's important to at least not blow up in the face of technically valid input.